replicatedhq/ship
# expand integration test for the github asset

What I Did
------------
Resolved https://github.com/replicatedhq/ship/issues/449 by demonstrating that the `github-slash` directory from the example contains the expected files.

How I Did it
------------
Added the example provided in the issue to an existing integration test.

How to verify it
------------
Look at the integration test diff, and ensure it runs and passes.

Description for the Changelog
------------



Picture of a Boat (not required but encouraged)
------------

![USS Jupiter (AC-3)](https://upload.wikimedia.org/wikipedia/commons/c/cb/USS_Jupiter_%28AC-3%29_at_Mare_Island_on_16_October_1913_%28NH_52365%29.jpg "USS Jupiter (AC-3)")
